package.json

```diff
@@ -41,9 +41,14 @@
     "mocha": "^9.1.3",
     "semver": "^7.3.5",
     "sinon": "^11.1.2",
-    "sinon-chai": "^3.7.0"
+    "sinon-chai": "^3.7.0",
+    "typescript": "^4.4.4",
+    "@typescript-eslint/parser": "^5.3.0",
+    "@typescript-eslint/eslint-plugin": "^5.3.0"
```

Comment:
Tip: If you add these via `npm i -D ...`, the dependencies get sorted correctly, avoiding unnecessary diffs when changing the dependencies using `npm` the next time.